src/codegen/mod.rs

```diff
@@ -2745,7 +2745,7 @@ mod utils {
     }
 
     pub fn fnsig_return_ty(ctx: &BindgenContext,
-                           sig: &super::FunctionSig)
+                           sig: &FunctionSig)
```

Comment:
This is only temporary until Gecko updates is rustc version anyway, should be <1week